lib/public/views/Logs/Details/LogTreeViewModel.js

```diff
@@ -137,7 +162,7 @@ export class LogTreeViewModel extends Observable {
              */
             const parseTreeNode = (node, level) => {
                 const { children, /** @type {Log} */ ...log } = node;
-                logTreeLeafs.push({ log, level });
+                logTreeLeafs.push({ log: log, level: level, showCopyUrlSuccessContent: false });
```

Comment:
When the key is the same as the variable containing the value, there is no need to specify it. Thus, for `log` and `level` it is ok to stay as `{ log, level, showCopyUrlSuccessContent: false }`